gensim/models/ldamodel.py

```diff
@@ -905,6 +905,28 @@ def get_document_topics(self, bow, minimum_probability=None):
         return [(topicid, topicvalue) for topicid, topicvalue in enumerate(topic_dist)
                 if topicvalue >= minimum_probability]
 
+    def get_static_topic(self, word_id, minimum_probability=None):
```

Comment:
I disagree. `0.0` is a specific value to use, whereas `None` stands for "use default". Different semantics.